src/hotspot/share/classfile/verifier.cpp

```diff
@@ -255,7 +255,7 @@ bool Verifier::verify(InstanceKlass* klass, bool should_verify_class, TRAPS) {
         // or one of it's superclasses, we're in trouble and are going
         // to infinitely recurse when we try to initialize the exception.
         // So bail out here by throwing the preallocated VM error.
```

Comment:
The comment looks wrong now as I think `THROW_MSG_` creates a new object.

Reply:
You're right - that's a good observation.  This code is hard to reach (tried to write a test case yesterday).  It might need a class file load hook for the java/lang/Error class, in which case, the whole VM is probably going to be useless.  But I will maintain throwing a preallocated instance.